src/scripts/updateFeatures.sjs

```diff
@@ -35,6 +45,48 @@ try {
 catch(err) {
   xdmp.log(err);
   res = err;
+  completed = false;
+}
+
+// email feature changes (only if there's actually a change, of course - no one likes spam :D)
+try {
+  var timestamp = fn.formatDateTime(fn.currentDateTime().add(xdmp.elapsedTime()), "[M01]/[D01]/[Y0001] [H01]:[m01]:[s01] ");
+
+  if (completed) {
+    var content = "Completed feature update at " + timestamp + "\n\n";
+    content += "Old features:\n";
+    for (var feature in oldFeatures) {
+      content += "\t- " + oldFeatures[feature] + "\n";
+    }
+    content += "\nNew features:\n";
+    for (var feature in input.features) {
+      content += "\t- " + input.features[feature] + "\n";
+    }
+
+    // because of how we insert feature arrays into MarkLogic,
+    // the order features appear in the array is preserved
+    if ("" + oldFeatures === "" + input.features) exit; // is this the right command to break out of this try/catch?
```

Comment:
as a matter of style, I'd prefer to put the next two lines within { }, to make it more visually clear that they depend on the `if`. The `exit` is easy to miss.